server/views/components/actions-card/template.njk

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-grid-column-one-third">
+  <div class="actions-list">
+    <ul class="govuk-list actions-list-list">
+      <span class="govuk-heading-m actions-list-header">Actions</span>
```

Comment:
I'm no expert in this stuff, but I wonder if the markup isnt quite right from a semantics point of view?

The `span`, especially with a class of `govuk-heading-m`, feels like it should be a heading? Perhaps a `h2` ?
And if we're in agreement that it is a heading, then I wonder if it should be outside of the `ul` ?

I know its a different use case and is rendering different content, but this is what we have for the Functional Skills sidebar (which currently sits where the proposed action bar will sit) (`functionalSkillsSidebar.njk`):
```
<div class="govuk-summary-card app-functional-skills-sidebar">
  <div class="govuk-summary-card__title-wrapper app-summary-card__title-wrapper">
    <h2 class="govuk-summary-card__title govuk-!-font-size-24">Current functional skills assessment scores</h2>
  </div>
  <div class="govuk-summary-card__content">

    {% if not functionalSkills.problemRetrievingData %}
      <table class="govuk-table govuk-!-margin-bottom-0" id="functional-skills-sidebar-table">
        <caption class="govuk-table__caption govuk-visually-hidden">Current functional skills and assessment information</caption>
        <thead class="govuk-table__head govuk-visually-hidden">
          <tr class="govuk-table__row">
            <th scope="col" class="govuk-table__header">Skill and assessment date</th>
            <th scope="col" class="govuk-table__header">Assessment level</th>
          </tr>
        </thead>
        <tbody class="govuk-table__body">
          {% for skill in functionalSkills.assessments %}
          <tr class="govuk-table__row">
            <th scope="row" class="govuk-table__header">
              {{ skill.type | formatFunctionalSkillType }}
              {% if skill.assessmentDate %}
              <br>
              <span class="govuk-body-s">Assessed on {{ skill.assessmentDate | formatDate('D MMMM YYYY') }}</span>
              {% endif %}
            </th>
            <td class="govuk-table__cell">
              {% if skill.grade %}{{ skill.grade }}{% else %}No functional skill assessment scores recorded in Curious{% endif %}
            </td>
          </tr>
          {% endfor %}
        </tbody>
      </table>
      <p class="govuk-body govuk-!-margin-top-2 govuk-!-display-none-print">
        <a class="govuk-link" href="../functional-skills" data-qa="view-all-functional-skills-button">View all <span class="govuk-visually-hidden">functional skills</span></a>
      </p>
    {% else %}
      <h3 class="govuk-heading-s" data-qa="functional-skills-sidebar-error-heading">We cannot show these details from Curious right now</h3>
      <p class="govuk-body">Reload the page or try again later. Other parts of this service may still be available.</p>
    {% endif %}
  </div>
</div>
```